### PR TITLE
Remove JudgehostRestriction feature

### PR DESCRIPTION
**Pull request by mvr320**
This is to trigger the pipeline, I think a quick grep for restrictions will only show other restrictions which are another type of restrictions.

### CODE REVIEW

**Comment by mvr320** [comment]
Changed URLs:
jury
jury_s_change-contest_s_-1
jury_s_judgehosts
jury_s_judgehosts_s_1
jury_s_judgehosts_s_1_s_activate
jury_s_config_s_check
jury_s_judgehosts_s_edit_s_multiple
jury_s_judgehosts_s_edit_s_multiple@include_hidden=0

File: webapp/src/Migrations/Version20210813195818.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    }
+
+    public function up(Schema $schema) : void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE judgehost DROP FOREIGN KEY judgehost_ibfk_1');
+        $this->addSql('DROP TABLE judgehost_restriction');
+        $this->addSql('DROP INDEX restrictionid ON judgehost');
```
Comment (thijskh) [inline webapp/src/Migrations/Version20210813195818.php:27]: What is the “change polltime”?

**Review by thijskh — CHANGES_REQUESTED** [review]
I think I’m missing a docs update.

**Comment by mvr320** [comment]
> 
> 
> I think I’m missing a docs update.

You mean the mention that the feature was removed? I'll add that.

File: webapp/src/Migrations/Version20210813195818.php (hunk 1 above)
Comment (mvr320) [reply webapp/src/Migrations/Version20210813195818.php:27]: That should not be there, its part of the doctrine:migrations:diff wanting the change the whole DB.

**Comment by thijskh** [comment]
I meant just removing the chapter in the manual. I think the manual is not the place for upgrade notes.

**Review by thijskh — APPROVED** [review]
Good work. Ready to merge when build passes and commits are squashed

File: webapp/src/Migrations/Version20210813195818.php (hunk 1 above)
Comment (nickygerritsen) [reply webapp/src/Migrations/Version20210813195818.php:27]: Our dev docs state how to fix this (set a car in .env.local)

**Pull request merged**